src/check-tsconfig.ts

```diff
@@ -1,10 +1,8 @@
-import { tsModule } from "./tsproxy";
 import * as tsTypes from "typescript";
-
 export function checkTsConfig(parsedConfig: tsTypes.ParsedCommandLine): void
 {
 	const module = parsedConfig.options.module!;
 
-	if (module !== tsModule.ModuleKind.ES2015 && module !== tsModule.ModuleKind.ESNext && module !== tsModule.ModuleKind.ES2020)
-		throw new Error(`Incompatible tsconfig option. Module resolves to '${tsModule.ModuleKind[module]}'. This is incompatible with rollup, please use 'module: "ES2015"' or 'module: "ESNext"'.`);
+	if (module !== tsTypes.ModuleKind.ES2015 && module !== tsTypes.ModuleKind.ESNext && module !== tsTypes.ModuleKind.ES2020)
+		throw new Error(`Incompatible tsconfig option. Module resolves to '${tsTypes.ModuleKind[module]}'. This is incompatible with rollup, please use 'module: "ES2015"' or 'module: "ESNext"'.`);
```

Comment:
all the `./tsproxy` -> `typescript` changes were from the original PR #135. I didn't touch them for now, just updated them.

Sounds like I may want to undo them per https://github.com/ezolenko/rollup-plugin-typescript2/pull/135#issuecomment-488047504 though

Reply:
Fixed this up. Undid the source changes from the original author and indeed needed to use `setTypescriptModule` and then everything started to work. Builds no longer have strange output as well.

Only weird thing was that importing and using `tsModule` from `tsproxy` didn't work -- it came out as `undefined` for some reason (possibly due to ESM semantics of static exports?). But it's using the same TS as the direct import, so I just used the direct import.